global/scripts/TopupPreprocessingAll.sh

```diff
@@ -55,13 +55,53 @@ opts_AddOptional '--topupconfig' 'TopupConfig' 'path' "topup config file"
 
 opts_AddOptional '--usejacobian' 'UseJacobian' 'true or false' "Whether to apply the jacobian of the gradient non-linearity distortion correction.  Irrelevant if --gdcoeffs=NONE (Has nothing to do with the jacobian of the TOPUP warp field)"
 
+opts_AddOptional '--phasezero' 'PhaseEncodeZero' 'image' "T2w assumed to be the negligible readout time, should be corrected for gradient non-linear distortions"
+
+opts_AddOptional '--phasezerobrainmask' 'SpinEchoPhaseEncodeZeroFSBrainmask' 'image' "T2w brainmask"
+
+opts_AddOptional '--phaseone2' 'PhaseEncodeOne2' 'image(s)' "first set of SE EPI images of 2nd phase dir: assumed to be the 'negative' PE direction"
+
+opts_AddOptional '--phasetwo2' 'PhaseEncodeTwo2' 'image(s)' "second set of SE EPI images of 2nd phase dir: assumed to be the 'positive' PE direction"
+
+opts_AddOptional '--scoutinbrain' 'ScoutInputBrainName' 'image' "scout input brain image: should be corrected for gradient non-linear distortions"
+
+opts_AddOptional '--scannerpatientposition' 'ScannerPatientPosition' 'string' "HFS (default), HFP"
+
+opts_AddOptional '--truepatientposition' 'TruePatientPosition' 'string' "HFS (default), HFSx, FFSx"
+
```

Comment:
If we say something is the default, let's actually have it as the default.

```suggestion
opts_AddOptional '--scannerpatientposition' 'ScannerPatientPosition' 'string' "HFS (default), HFP" 'HFS'

opts_AddOptional '--truepatientposition' 'TruePatientPosition' 'string' "HFS (default), HFSx, FFSx" 'HFS'

```